### DISCUSSION TITLE
Adjustable radio transcript font size to support reading from across-the-room

### DISCUSSION THREAD

**Discussion by interi-mhickman** [f1multiviewer/issue-tracker#577] [Feature requests]
I use multi viewer to display stats, in-car cameras, and radio transcripts while watching the race on TV. Requesting the ability to increase only the radio transcript panel / window font size to enable it to be readable from across the room. I currently use Ctrl-+ to zoom the text, but this zooms all content across all Multiviewer windows.

**Comment by jeppeolesen** [comment]
Closing this as a duplicate and redirecting discussion about the zoom features to #187
